examples/travel-lb.amp.html

```diff
@@ -0,0 +1,197 @@
+    background-repeat: no-repeat;
+    background-size: 25px 25px;
+  }
+
+  #returning {
+    border-left: 0;
+  }
+
+  @media (min-width: 480px) {
+    .fields {
+      flex-flow: row nowrap;
+    }
+    #departing, #returning {
+      width: 50%;
+    }
+  }
+
+
+  .lightbox-close {
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: 10;
+    width: 20px;
+    height: 20px;
+    padding: 20px 15px;
+    cursor: pointer;
+    box-sizing: content-box;
+    border: 0;
+    background-color: transparent;
+  }
+  .lightbox-close::before {
+    content: '';
+    display: block;
+    width: 20px;
+    height: 20px;
+    background-image: url('data:image/svg+xml,<svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 12"><path fill="#fff" fill-rule="evenodd" d="M11.53,0.47 C11.237,0.177 10.762,0.177 10.469,0.47 L5.999,4.94 L1.529,0.47 C1.236,0.177 0.761,0.177 0.468,0.47 C0.175,0.763 0.175,1.238 0.468,1.531 L4.938,6.001 L0.468,10.471 C0.175,10.764 0.175,11.239 0.468,11.532 C0.761,11.825 1.236,11.825 1.529,11.532 L5.999,7.062 L10.469,11.532 C10.762,11.825 11.237,11.825 11.53,11.532 C11.823,11.239 11.823,10.764 11.53,10.471 L7.06,6.001 L11.53,1.531 C11.823,1.238 11.823,0.763 11.53,0.47 L11.53,0.47 Z"/></svg>');
+    background-repeat: no-repeat;
+    background-size: 100%;
+  }
+
+  .highlight-square {
+    content: '';
+    background-color: #20bf7c;
+    width: 5px;
+    height: 5px;
+    position: absolute;
+    bottom: 5px;
+    margin: auto;
+    left: 0;
+    right: 0;
+  }
+  </style>
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async custom-element="amp-lightbox" src="https://cdn.ampproject.org/v0/amp-lightbox-0.1.js"></script>
+  <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+  <script async custom-element="amp-date-picker" src="https://cdn.ampproject.org/v0/amp-date-picker-0.1.js"></script>
+  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+<div class="search">
+  <p>Choose your travel dates</p>
+  <div class="fields">
+    <input on="tap:lb" placeholder="Departing" id="departing"/>
```

Comment:
let's be explicit and call `lb.open`

Reply:
Actually there's bug. You can't call open on a lightbox until it has been opened already #13755